app/config/schema/rooms.proto

```diff
@@ -93,6 +94,11 @@ message Room {
     // The page to redirect to
     // Required for redirect rooms.
     string route = 16;
+
+    // CHATSTREAM ROOMS ONLY
+    // Options related to the bosh service
+    // Required for chat streaming rooms.
+    string boshUrl = 17;
```

Comment:
Should this be in `Room`? Since `boshUrl` is always the same, does it need to be part of the room config? Can we make it part of the general config?

Reply:
Good question: I was just following the precedent, but it could certainly be generalized.